packages/salesforcedx-utils-vscode/src/services/types/FileInfo.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export type FileInfo = { type: string; fullName: string; filePath?: string };
```

Comment:
what is type here?

Reply:
I just created it.  I wasn't sure what to do with the types here.  I get StatusOutputRow[] back from STL.  I guess I can pass that straight to PersistentStorageService...

Reply:
Oh wait, maybe I'll export the StatusOutputRow type from SourceTrackingService...